PixelHunter1995/Utilities/Polygon.cs

```diff
@@ -183,7 +184,7 @@ private bool IsVertexAnEar(int i)
 
         /// <summary>
         /// Checks if point is contained in the polygon.
-        /// Taken from https://stackoverflow.com/a/2922778
+        /// Taken from https://wrf.ecse.rpi.edu/Research/Short_Notes/pnpoly.html
```

Comment:
This one actually has a license, which we don't follow. However, the code seems to only be lightly based on the code in the link. Maybe reformulate to "lightly based on" or something, instead of writing "taken" and then not actually following the license...